include/wx/peninfo.h

```diff
@@ -0,0 +1,145 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/peninfo.h
```

Comment:
I'm not sure what's going on here, but could it be that `wx/peninfo.h` is included from `wx/pen.h` before these constants are defined in it?

In any case, it is definitely a bad idea to have inclusions in both directions between these files. So either `wxPenInfo` itself should just to into the existing `wx/pen.h` or the styles need to be moved into `wx/peninfo.h` too.